ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/editor/EditorAgentImpl.java

```diff
@@ -574,6 +597,51 @@ private void restoreInitEditor(final VirtualFile file, final OpenEditorCallback
         finalizeInit(file, callback, editor, editorProvider);
     }
 
+    @Override
+    public void onSelectionChanged(SelectionChangedEvent event) {
+        final String isLinkedWithEditor = preferencesManager.getValue(LinkWithEditorAction.LINK_WITH_EDITOR);
+        if (isNullOrEmpty(isLinkedWithEditor) || !Boolean.parseBoolean(isLinkedWithEditor)) {
```

Comment:
Why? If value of the `LinkWithEditorAction.LINK_WITH_EDITOR` wasn't set we'll have empty result of the `preferencesManager.getValue(LinkWithEditorAction.LINK_WITH_EDITOR);`